curator/style.css

```diff
@@ -92,6 +92,63 @@ Tags: one-column, custom-colors, custom-menu, custom-logo, editor-style, feature
 	padding: calc(0.667em + 2px) calc(1.333em + 2px);
 }
 
+/*
+ * Link Styles
+ */
+
+a {
+	padding: .4rem 0;
```

Comment:
I think we should give links a bit of an offset: 

Current | Suggested
--- | ---
<img width="149" alt="Screen Shot 2022-05-09 at 8 19 15 AM" src="https://user-images.githubusercontent.com/5375500/167408428-8ef70717-202a-44a5-945f-eb7e007d40bf.png"> | <img width="146" alt="Screen Shot 2022-05-09 at 8 19 05 AM" src="https://user-images.githubusercontent.com/5375500/167408537-e9c57746-2b40-4fd9-9c01-4c1deb3b8017.png">

```suggestion
	padding: .4rem 0;
	text-underline-offset: .1rem;
```

Reply:
Updated the style.